docs/windows_developer_guide.md

```diff
@@ -10,7 +10,7 @@ Following the steps below requires 40GB of free disk space.
 
 ## Install git
 Download from https://github.com/git-for-windows/git/releases/download/v2.47.1.windows.2/Git-2.47.1.2-64-bit.exe
-Or run command in windows cmd.exe terminal:
+Or run command in windows cmd.exe terminal as Administrator:
```

Comment:
I think we don't need to maintain instruction for git installation. Maybe it would be sufficient to inform about git installation and point to https://git-scm.com/downloads/win
This section could be called "Pull OpenVINO Model Server source" with info to install git and git clone command (admin permissions not required)